packages/ottehr-components/lib/components/CustomContainerFactory.tsx

```diff
@@ -73,7 +73,7 @@ export const CustomContainer: FC<ContainerProps> = ({
     if (logoutHandler !== undefined) {
       logoutHandler();
     } else {
-      logout({ returnTo: 'https://ottehr.com' });
+      logout({ returnTo: 'https://telemed.ottehr.com/location/ak/in-person/prebook' });
```

Comment:
why is this set in the code? not every user needs this url

Reply:
should I change it to https://telemed.ottehr.com ?

Reply:
yes